core/components/atoms/input/Input.tsx

```diff
@@ -206,8 +206,11 @@ export const Input = React.forwardRef<HTMLInputElement, InputProps>((props, forw
   });
 
   const trigger = (
-    <div className={rightIconClass}>
-      <Icon name={'info'} size={sizeMapping[size]} />
+    <div
+      className={rightIconClass} // eslint-disable-next-line jsx-a11y/no-noninteractive-tabindex
+      tabIndex={0}
+    >
+      <Icon name={'info'} size={sizeMapping[size]} className="Input-rightIcon" />
```

Comment:
Instead of defining new class `Input-rightIcon` we can re-use the `Input-icon--right` class which is not used anywhere

Reply:
The class has to be used otherwise the code is breaking